test/test_prototype_transforms_consistency.py

```diff
@@ -480,16 +501,22 @@ def test_signature_consistency(config):
         )
 
     extra = prototype_params.keys() - legacy_params.keys()
-    extra_without_default = {param for param in extra if prototype_params[param].default is not inspect.Parameter.empty}
+    extra_without_default = {
+        param
+        for param in extra
+        if prototype_params[param].default is inspect.Parameter.empty
+        and prototype_params[param].kind not in {inspect.Parameter.VAR_POSITIONAL, inspect.Parameter.VAR_KEYWORD}
```

Comment:
The check was faulty in that it also took `*args` or `**kwargs` as required. These are now explicitly excluded.